internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -284,6 +285,20 @@ var _ = Describe("Validate clusterlogforwarderspec", func() {
 			clfStatus = &loggingv1.ClusterLogForwarderStatus{}
 			extras = map[string]bool{}
 
+			cloudWatchSecret = &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "mytestsecret",
+					Namespace: constants.OpenshiftNS,
+				},
+				Data: map[string][]byte{},
+			}
+			cloudWatchSecret.Data[constants.AWSSecretAccessKey] = []byte{0, 1, 2}
```

Comment:
This can be inlined during the secret initialization

Reply:
done